### ISSUE TITLE
Palpable thrill

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10497]
**Preferred term label:**
Palpable thrill

**Synonyms**
pulsating or vibrating mass, often due to an arteriovenous malformation

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
AVM on an extremity
Parkes Weber syndrome

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
if we say just "palpable thrill" it would usually mean over the heart, but this seems to be over an extremity? Please clarify.

**Comment by MickeySegal** [comment]
I'm fine with adding "on an extremity" - all our uses are for AVMs.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001055
- label: Palpable thrill on extremity
- parents: [Abnormal arterial physiology (HP:0025323)](https://hpo.jax.org/app/browse/term/HP:0025323)
- pmid: 
- def: A vibration felt upon palpation of a blood vessel over an extremity.
- comment: This feature can be due to an arteriovenous malformation.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10497
- synonyms:
